core/domain/voiceover_services.py

```diff
@@ -144,6 +129,36 @@ def get_all_language_accent_codes_for_voiceovers(
     return language_codes_mapping
 
 
+def create_entity_voiceovers_model_instance(
+    entity_voiceovers_instance: voiceover_domain.EntityVoiceovers
+) -> voiceover_models.EntityVoiceoversModel:
+    """Creates an entity voiceovers model instance.
```

Comment:
Maybe add a note to explain that this just returns the model instance, but does not save it to the datastore.